src/zenml/integrations/evidently/data_validators/evidently_data_validator.py

```diff
@@ -52,60 +49,132 @@
 logger = get_logger(__name__)
 
 
-profile_mapper = {
-    "datadrift": DataDriftProfileSection,
-    "categoricaltargetdrift": CatTargetDriftProfileSection,
-    "numericaltargetdrift": NumTargetDriftProfileSection,
-    "dataquality": DataQualityProfileSection,
-    "classificationmodelperformance": ClassificationPerformanceProfileSection,
-    "regressionmodelperformance": RegressionPerformanceProfileSection,
-    "probabilisticmodelperformance": ProbClassificationPerformanceProfileSection,
-}
-
-dashboard_mapper = {
-    "dataquality": DataQualityTab,
-    "datadrift": DataDriftTab,
-    "categoricaltargetdrift": CatTargetDriftTab,
-    "numericaltargetdrift": NumTargetDriftTab,
-    "classificationmodelperformance": ClassificationPerformanceTab,
-    "regressionmodelperformance": RegressionPerformanceTab,
-    "probabilisticmodelperformance": ProbClassificationPerformanceTab,
-}
-
-
-def get_profile_sections_and_tabs(
-    profile_list: Optional[Sequence[str]],
-    verbose_level: int = 1,
-) -> Tuple[List[ProfileSection], List[Tab]]:
-    """Get the profile sections and dashboard tabs for a profile list.
+def get_metric_class_from_mapping(metric: str) -> Union[Metric, MetricPreset]:
+    """Get the Evidently metric or metric preset class from a string.
 
     Args:
-        profile_list: List of identifiers for Evidently profiles.
-        verbose_level: Verbosity level for the rendered dashboard. Use
-            0 for a brief dashboard, 1 for a detailed dashboard.
+        metric: Name of the metric or metric preset.
 
     Returns:
-        A tuple of two lists of profile sections and tabs.
+        The Evidently metric or metric preset class.
 
     Raises:
-        ValueError: if the profile_section is not supported.
+        ValueError: If the metric or metric preset is not a valid Evidently metric or metric preset.
     """
```

Comment:
I need advice here. Is this the right way to handle loading the classes from the string values? Earlier, we used a map to link the strings and their respective classes.